### PR TITLE
Read master dataset

### LINKED ISSUE #70: Read the broadcast datasets
### Feature you want to implement
Fetch the datasets that are broadcast to artiq master.

### How the feature is implemented
Artiq master will hold the datasets, so we will use RPC to call the getter method of the dataset DB.

### Additional context
Still investigating...

### PR DESCRIPTION
**Pull request by kangz12345**
This closes #70.

`get_master_dataset(key)` returns the dataset in the artiq master's dataset DB, searching the value for the given key.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -448,6 +449,20 @@ async def list_result_directory() -> List[int]:
...
+@app.get("/dataset/master/")
+async def get_master_dataset(key: str) -> Union[int, float, List]:
+    """Returns the dataset broadcast to artiq master.
+    
+    Args:
+        key: The key of the target dataset.
+    """
+    remote = get_client("master_dataset_db")
+    array = remote.get(key)
+    if isinstance(array, np.ndarray):
```
Comment (BECATRUE) [inline main.py:461]: It is just a question!
When isn't the `array` a numpy array? The case it is a number?

**Review by BECATRUE — APPROVED** [review]
I'm sorry that I checked too late. It looks cool!

File: main.py (hunk 1 above)
Comment (kangz12345) [reply main.py:461]: The dataset in artiq does not restrict the type of the values in it. It can be anything in principle!

Comment (BECATRUE) [reply main.py:461]: Oh, I see! Thanks👍

**Pull request merged**